megamek/src/megamek/client/generator/TeamLoadoutGenerator.java

```diff
@@ -652,15 +652,62 @@ public static ReconfigurationParameters generateParameters(
 
     // region Imperative mutators
     private static void setACImperatives(Entity e, MunitionTree mt, ReconfigurationParameters rp) {
+        setAC5Imperatives(e, mt, rp);
+        setAC10Imperatives(e, mt, rp);
         setAC20Imperatives(e, mt, rp);
     }
 
+    private static int getACWeaponCount(Entity e, String size) {
+        return (int) e.getWeaponList().stream()
+                .filter(w -> w.getName().toLowerCase().contains("ac") && w.getName().contains(size)).count();
```

Comment:
Can we have a new-line at `.contains` and `.count`, it might not be strictly necessary from a Style Guide perspective, but it's my experience that it makes the stream much easier to parse at a glance